doc/api/async_context.md

```diff
@@ -446,6 +446,9 @@ added:
   - v14.8.0
   - v12.19.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/42177
+    description: Changed the default when `thisArg` is not provided to use `this` from the caller.
```

Comment:
```suggestion
    description: Changed the default when `thisArg` is undefined to use `this` from the caller.
```

Reply:
@rochdev does this suggestion loos good to you?

Reply:
LGTM, feel free to commit it in both places.